homeworks/sasha.demidenko_SashaDemi/GamesProject/css/games.css

```diff
@@ -29,7 +29,7 @@
 .games_header--ul {
   display: inline-grid;
   justify-items: end;
-  grid-template: 1fr/ 78px 78px;
+  grid-template: 1fr/ 78px 78px 78px;
```

Comment:
If you are not sure about grid, maybe just use flex? It is easy to understand and just perfect for single-row/column containers like this.